VisionFive/Armbian/README_zh.md

```diff
@@ -0,0 +1,321 @@
+122) es_UY.UTF-8              291) tcy_IN.UTF-8
+123) es_VE.UTF-8              292) te_IN
+124) et_EE.UTF-8              293) tg_TJ.UTF-8
+125) et_EE.UTF-8              294) the_NP
+126) eu_ES.UTF-8              295) th_TH.UTF-8
+127) eu_FR.UTF-8              296) ti_ER
+128) fa_IR                    297) ti_ET
+129) ff_SN                    298) tig_ER
+130) fi_FI.UTF-8              299) tk_TM
+131) fil_PH                   300) tl_PH.UTF-8
+132) fo_FO.UTF-8              301) tn_ZA
+133) fr_BE.UTF-8              302) to_TO
+134) fr_CA.UTF-8              303) tpi_PG
+135) fr_CH.UTF-8              304) tr_CY.UTF-8
+136) fr_FR.UTF-8              305) tr_TR.UTF-8
+137) fr_LU.UTF-8              306) ts_ZA
+138) fur_IT                   307) tt_RU
+139) fy_DE                    308) tt_RU@iqtelif
+140) fy_NL                    309) tt_RU@iqtelif
+141) ga_IE.UTF-8              310) ug_CN
+142) gbm_IN                   311) ug_CN@latin
+143) gd_GB.UTF-8              312) ug_CN@latin
+144) gez_ER                   313) uk_UA.UTF-8
+145) gez_ER@abegede           314) unm_US
+146) gez_ER@abegede           315) ur_IN
+147) gez_ET                   316) ur_PK
+148) gez_ET@abegede           317) uz_UZ.UTF-8
+149) gez_ET@abegede           318) uz_UZ@cyrillic
+150) gl_ES.UTF-8              319) uz_UZ@cyrillic
+151) gu_IN                    320) ve_ZA
+152) gv_GB.UTF-8              321) vi_VN
+153) hak_TW                   322) wa_BE.UTF-8
+154) ha_NG                    323) wae_CH
+155) he_IL.UTF-8              324) wal_ET
+156) hif_FJ                   325) wo_SN
+157) hi_IN                    326) xh_ZA.UTF-8
+158) hne_IN                   327) yi_US.UTF-8
+159) hr_HR.UTF-8              328) yo_NG
+160) hsb_DE.UTF-8             329) yue_HK
+161) ht_HT                    330) yuw_PG
+162) hu_HU.UTF-8              331) zgh_MA
+163) hy_AM                    332) zh_CN.UTF-8
+164) ia_FR                    333) zh_HK.UTF-8
+165) id_ID.UTF-8              334) zh_SG.UTF-8
+166) ig_NG                    335) lzh_TW
+167) ik_CA                    336) zh_TW.UTF-8
+168) is_IS.UTF-8              337) zu_ZA.UTF-8
+169) it_CH.UTF-8              338) Skip generating locales
+Please enter your choice:338
+root@visionfive:~# uname -a
+Linux visionfive 6.1.82-edge-starfive #2 SMP Fri Mar 15 18:27:50 UTC 2024 riscv64 riscv64 riscv64 GNU/Linux
+root@visionfive:~# cat /etc/os-release 
+PRETTY_NAME="Armbian_community 24.5.0-trunk.278 noble"
+NAME="Ubuntu"
+VERSION_ID="24.04"
+VERSION="24.04 (Noble Numbat)"
+VERSION_CODENAME=noble
+ID=ubuntu
+ID_LIKE=debian
+HOME_URL="https://github.com/armbian/build"
+SUPPORT_URL="https://community.armbian.com/"
+BUG_REPORT_URL="https://github.com/armbian/community/issues"
+PRIVACY_POLICY_URL="https://duckduckgo.com/"
+UBUNTU_CODENAME=noble
+LOGO="armbian-logo"
+ARMBIAN_PRETTY_NAME="Armbian_community 24.5.0-trunk.278 noble"
+root@visionfive:~# 
+
+```
+
+## 测试判定标准
+
+测试成功：实际结果与预期结果相符。
+
+测试失败：实际结果与预期结果不符。
+
+## 测试结论
+
+测试部分成功。
```

Comment:
^Same as above. Did something failed or is this a typo?

Reply:
This is due to the u-boot have a issue at that time, since now the fix pr is merged, maybe a retest is needed later?

Reply:
Related pr: https://github.com/starfive-tech/u-boot/pull/31